### ISSUE TITLE
ReadValue returning a nan gives an error in Monitoring

### ISSUE THREAD

**Issue by ograsdijk** [js216/CeNTREX#76]
When a driver returns a single np.nan from the ReadValue function the nan is pushed to the data_queue and plots_queue. The monitoring class does not allow for a single float in the following lines
```
if not isinstance(data, type(None)):
                    # display the data in a tkinter variable
                    try:
                        if dev.config["slow_data"]:
                            formatted_data = [np.format_float_scientific(x, precision=3) for x in data]
                        else:
                            formatted_data = [np.format_float_scientific(x, precision=3) for x in data[0][0,:,0]]
                    except TypeError as err:
                        logging.warning("Warning in Monitoring: " + str(err))
                        continue
```

**Comment by js216** [comment]
Fixed in 2cc447b by only processing data that's in a list.

`ReadValue()` should return a `list` of values; usually, the first element is the time, and the following entries are the quantities measured by the device. Thus I have changed

    if not isinstance(data, type(None)):

into

    if isinstance(data, list):

**Comment by js216** [comment]
Also, f2bd11e makes sure the PXIe5171 returns a list, not a tuple.
